.infrastructure/todoapp-pod.yml

```diff
@@ -21,3 +30,37 @@ spec:
         port: 8080
       initialDelaySeconds: 5
       periodSeconds: 5
+
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: todoapp-2
+  namespace: todoapp
+  labels:
+    app: todolist
+spec:
+  containers:
+  - name: todoapp
+    image: ikulyk404/todoapp:3.0.0
+    ports:
+    - containerPort: 8080
+    env:
+    - name: ENVIRONMENT
+      value: "production"
+    - name: DEBUG
+      value: "false"
+    - name: API_URL
+      value: "http://todoapp-clusterip"
+    livenessProbe:
+      httpGet:
+        path: api/health
+        port: 8080
+      initialDelaySeconds: 60
+      periodSeconds: 5
+    readinessProbe:
+      httpGet:
+        path: api/ready
+        port: 8080
+      initialDelaySeconds: 5
+      periodSeconds: 5
```

Comment:
Add Eof(End-of-file), need to add a blank line at the end
![image](https://github.com/user-attachments/assets/66e3db2e-935c-4d94-935c-e844bd91d50d)